### DISCUSSION TITLE
후기 모임 - 북클럽 세번째 <함께 자라기>, 2022-11-17

### DISCUSSION THREAD

**Discussion by jongfeel** [silqwer/book-club#5] [Show and tell]
담백하게 사진만 올려 봅니다.

언젠가의 추억의 사진이 될 것 같은 느낌이네요.
폭풍 전야의 아무것도 몰랐던 우리들의 모습.

![image](https://user-images.githubusercontent.com/17442457/203683716-9ce6d211-6634-4b2d-ac7f-79072dc3f6f5.png)

![image](https://user-images.githubusercontent.com/17442457/203683765-d9481d6f-5ff3-44d2-956f-112796b1b6e1.png)

**Comment by silqwer** [comment]
첫 번째 사진에서 전 눈을 감았군욤 😆
